### ISSUE TITLE
CRT Shaders are not applied to PSX, Saturn, Dreamcast, etc...

### ISSUE THREAD

**Issue by antonyho** [dragoonDorise/EmuDeck#632]
2D and 3D CRT shaders enabled through installation step, or Quick Settings are not working. CRT shader setting was not added to 16bit console emulator config files.

**Comment by Godsbane** [comment]
They work, it's been tested over and over.

You need to use the base folders. If you use subfolders it does not work.

If you need support please join the discord.

**Comment by antonyho** [comment]
I have tested to use the Quick Settings to turn on and off CRT shader, on both 2D and 3D retro engines. And tried waited until the notification to dismiss itself. However, the line of 'video_filter' in config files is always missing.
For example, in my `snes.cfg` the line of `video_filter` is always missing. 

Both lines are missing when I switch the filter on and off.
Neither
`video_filter = ED_RM_LINE`
nor
`video_filter = "/app/lib/retroarch/filters/video/Normal4x.filt"`

I have checked all `cfg` files for the retro engines I used. They are all missing the `video_filter` line. I didn't know how did the shell script function failed to add the `video_filter` lines. I can confirm that `video_shader_enable` and `video_smooth` lines are there.
